corehq/apps/fixtures/models.py

```diff
@@ -330,30 +330,27 @@ def soft_delete(self):
         self.save()
         CSQLFixtureExpressionLog.objects.create(
             expression=self,
-            action=CSQLFixtureExpressionLog.DELETE,
+            action=CSQLFixtureExpressionLog.Action.DELETE.value,
         )
 
 
 class CSQLFixtureExpressionLog(models.Model):
-
-    CREATE = 'create'
-    DELETE = 'delete'
-    UPDATE = 'update'
+    class Action(models.TextChoices):
+        CREATE = 'CR', _('Create')
+        DELETE = 'DE', _('Delete')
+        UPDATE = 'UP', _('Update')
 
     expression = ForeignKey(CSQLFixtureExpression, on_delete=models.CASCADE)
     date = models.DateTimeField(auto_now_add=True)
-    action = models.CharField(max_length=16, choices=(
-        (UPDATE, _('Updated')),
-        (CREATE, _('Created')),
-        (DELETE, _('Deleted')),
-    ), null=False)
+    action = models.CharField(max_length=2, choices=Action.choices, null=False)
     name = models.CharField(max_length=64, default='')
-    csql = models.CharField(max_length=2000, default='')
+    csql = models.TextField(default='')
 
 
 @receiver(post_save, sender=CSQLFixtureExpression)
 def after_save(sender, instance, created, **kwargs):
-    updated_or_created = CSQLFixtureExpressionLog.CREATE if created else CSQLFixtureExpressionLog.UPDATE
+    updated_or_created = (CSQLFixtureExpressionLog.Action.CREATE.value if created
+                          else CSQLFixtureExpressionLog.Action.UPDATE.value)
```

Comment:
nit: you actually don't need to call `.value` on these - you can generally treat them as a string (corresponding to the value)